Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -403,31 +403,39 @@ def getJobOutput(self, jobID, _localDir=None):
 
     if not self.useLocalSchedd:
       iwd = None
-      if pathToResult:
-        iwd = os.path.join(self.workingDirectory, pathToResult)
-      else:
-        status, stdout_q = commands.getstatusoutput('condor_q %s %s -af SUBMIT_Iwd' % (self.remoteScheddOptions,
-                                                                                       condorID))
-        self.log.verbose('condor_q:', stdout_q)
-        if status != 0:
-          return S_ERROR(stdout_q)
-        if self.workingDirectory in stdout_q:
-          iwd = stdout_q
 
+      # TOREMOVE: once v7r0 will mainly be used, remove the following block that was only useful
+      # when path to output was not deterministic
+      status, stdout_q = commands.getstatusoutput('condor_q %s %s -af SUBMIT_Iwd' % (self.remoteScheddOptions,
+                                                                                     condorID))
+      self.log.verbose('condor_q:', stdout_q)
+      if status == 0 and self.workingDirectory in stdout_q:
+        iwd = stdout_q
+
+      # Use the path extracted from the pilotID
       if iwd is None:
-        return S_ERROR("Failed to find condor job %s" % condorID)
+        iwd = os.path.join(self.workingDirectory, pathToResult)
 
       try:
-        os.makedirs(iwd)
+        mkDir(iwd)
```

Comment:
I think (or somewhere else in this block)
```suggestion
        mkDir(iwd)
        pathToResult = iwd
```
otherwise below the pathToResult from the pilot stamp is used, which might be different.